esileapclient/osc/v1/lease.py

```diff
@@ -133,6 +133,16 @@ def get_parser(self, prog_name):
             required=False,
             help="Show all leases relevant to an offer owner "
                  "by the owner's project_id.")
+        parser.add_argument(
+            '--resource-type',
+            dest='resource_type',
+            required=False,
+            help="Show all leases with given resource-type.")
```

Comment:
i can nit here and say that you use a dash 'resource-type' and in the help text here but for the project_id you use underscore